packages/common/src/search/_internal/portalSearchUsers.ts

```diff
@@ -120,18 +120,18 @@ async function searchPortal(
  * Convert an Item to a IHubSearchResult
  * Fetches the includes and attaches them to the item
  * @param item
- * @param includes
+ * @param include
  * @param requestOptions
  * @returns
  */
 async function userToSearchResult(
   user: IUser,
-  includes: string[] = [],
+  include: string[] = [],
```

Comment:
renamed to match the IHubSearchOptions property name b/c the mismatch was confusing me